src/models/User.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
+ * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
+ */
+package models;
+
+import java.sql.Date;
+
+/**
+ *
+ * @author DELL
+ */
+public class User {
+    private Integer id;
+    private String fName;
+    private String lName;
+    private String phoneNum;
+    private String uEmail;
+    private String password;
+    private Date createdDate;
+    private Date modifiedDate;
+    private boolean status;
+
```

Comment:
please create a seperate clas for Auditable fields i.e id, createdDate,modifiedDate, status and also add two more fields createdBy and modifiedBy